Story/Episode 3/np_010851.csv

```diff
@@ -0,0 +1,53 @@
+11_voice_NPC_5085_104#0,"""About ARKS... Let's start from the Beginning of Arks."""
+text01002#0,"""The name \""ARKS\"" was given to the team that investigates the various planets in the Universe."""
+text01003#0,"""The purpose of ARKS is not simply conducting investigations. They are also responsible for annihilating Darkers or any enemies corrupted by Darkers."""
+text01004#0,"""ARKS was first created by a Civilization known as the Photoners."""
+text01005#0,"""No... ARKS was created by a Being revered as a God by Photoners known as \""Xion\"". """
+text01006#0,"""All for the purpose of defeating Darkers, and act as the Guardians of the Universe"""
+text01007#0,"""Over the years, the objectives of ARKS seemed to have veered off course. Somehow, I managed to guide it back. """
+11_voice_NPC_5085_100#0,"""Exploring the Universe, Investigate Planets. Encountering any Darkers, fight and defeat them. That is ARKS."""
```

Comment:
"Explore the universe, investigate planets. When encountering Darkers, fight and defeat them."